automation-tests/.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
+// Use IntelliSense to learn about possible attributes.
+// Hover to view descriptions of existing attributes.
+// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug All Automation Tests",
```

Comment:
How can I see this configuration?

Reply:
I've been opening up `code automation-tests` separately, in a separate instance of VS Code.  I need to look into adding a config to our main project, so that this sub-project's can run.